### PR TITLE
Also mention airflow 2 only in readme template

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragement file, named `{pr_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by josh-fell** [comment]
Did you want to update `PIP requirements` to `Requirements` like in the other template too?

**Comment by potiuk** [comment]
:facepalm: :facepalm:

**Comment by potiuk** [comment]
Done. BTW I removed `PIP` actually because pip maintainers are very "touchy" about `pip` casing BTW..

CC: @uranusjr -> just want to be nice

**Review by josh-fell — APPROVED** [review]

File: dev/provider_packages/PROVIDER_README_TEMPLATE.rst.jinja2 [hunk 1]
```diff
@@ -44,15 +44,16 @@ in the `documentation <https://airflow.apache.org/docs/{{ PACKAGE_PIP_NAME }}/{{
 Installation
 ------------
 
-You can install this package on top of an existing Airflow 2.2+ installation via
+You can install this package on top of an existing Airflow 2 installation (see ``Requirements`` below)
```
Comment (josh-fell) [inline dev/provider_packages/PROVIDER_README_TEMPLATE.rst.jinja2]:
```suggestion
You can install this package on top of an existing Airflow 2 installation (see ``Requirements`` below
```

Comment (josh-fell) [reply dev/provider_packages/PROVIDER_README_TEMPLATE.rst.jinja2]: I didn't catch that in the index template either. 🤦

**Pull request merged**